### ISSUE TITLE
George Paci's lore patch

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11319]
|<img alt="jelkner's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| jelkner reported|
|-|-|
|Trac ID|trac#443|
|Type|enhancement|
|Created|2003-12-22 06:54:57Z|

Attachments:

* [gpaci_lore_patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/ac3/ac3e7b007d7ab0ba379faa8ab62d9da35c5444f4/b1ae480f905c95dab8d18e5c68f3ec750f555ef7) (33834 bytes) - added by jelkner on 2003-12-22 06:54:57Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__443 443
type__enhancement enhancement
reporter__jelkner jelkner
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1072076097000000 1072076097000000
changetime__1470694695811508 1470694695811508
version__ 
owner__ 
cc__spiv cc__moshez cc__jelkner
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="jelkner's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|jelkner commented|
|-|-|

```
#!html
<pre>
Please email me at: jeff@... to let me know how this goes.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
jelkner committed this
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub commented|
|-|-|

In changeset 88d14580e6ff4da4be8d84440852b365d427aa31
```
#!CommitTicketReference repository="" revision="88d14580e6ff4da4be8d84440852b365d427aa31"
Merge pull request #443 from twisted/8703-rodrigc-getargspec-py3

Author: rodrigc
Reviewer: Lukasa
Fixes: #8703

Do not use inspect.getargspec() in twisted/names/dns.py on Python 3
```
